server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/GlobalOrdinalsStringTermsAggregator.java

```diff
@@ -648,7 +647,6 @@ void updateBucket(OrdBucket spare, long globalOrd, long bucketOrd, long docCount
             spare.globalOrd = globalOrd;
             spare.bucketOrd = bucketOrd;
             spare.docCount = docCount;
-            otherDocCount += docCount;
```

Comment:
This was causing inaccurate doc counts when `shard_min_doc_count` was more than 1.